src/ripple/shamap/impl/SHAMap.cpp

```diff
@@ -73,6 +73,7 @@ SHAMap::snapShot (bool isMutable) const
 
     newMap.seq_ = seq_ + 1;
     newMap.root_ = root_;
+    newMap.backed_ = backed_;
```

Comment:
One could argue that `ledgerSeq_` should be copied if the snapShot isn't mutable and not if it is, but it won't make any real difference. I don't particularly like the idea of spreading the unusual logic of this function over two functions.